source/extensions/common/wasm/v8/v8.cc

```diff
@@ -515,6 +566,18 @@ std::unique_ptr<Global<T>> V8::registerHostGlobalImpl(absl::string_view moduleNa
   return proxy;
 }
 
+std::unique_ptr<Global<Word>> V8::registerHostWordGlobalImpl(absl::string_view moduleName,
+                                                             absl::string_view name,
+                                                             Word initialValue) {
+  ENVOY_LOG(trace, "[wasm] registerHostGlobal(\"{}.{}\", {})", moduleName, name, initialValue);
+  auto value = wasm::Val::make(static_cast<uint32_t>(initialValue.u64));
```

Comment:
You could change this line to `auto value = makeVal(initialValue);` in the version above, and then we wouldn't need to have a specialized version for `Word`.

Reply:
done.